backend/services/services.py

```
@@ -0,0 +1,213 @@
+#all helper functions related to post, will be here
+from uuid import uuid4
```

Ruff (0.8.2)

`uuid.uuid4` imported but unused

Remove unused import: `uuid.uuid4`

(F401)